core/src/main/scala/kafka/server/DynamicBrokerConfig.scala

```diff
@@ -35,6 +35,7 @@ import org.apache.kafka.common.network.{ListenerName, ListenerReconfigurable}
 import org.apache.kafka.common.security.authenticator.LoginManager
 import org.apache.kafka.common.utils.{ConfigUtils, Utils}
 
+import java.util.concurrent.CopyOnWriteArrayList
```

Comment:
nit: Could we either move this one next to the other `java.util.*` imports or move the others down here?

Reply:
I moved it up to the others. hope that looks okay.